### PR TITLE
nl-NL: Fix many translations

### PR DESCRIPTION
**Pull request by Gymnasiast**
Mostly corrections for vanilla translations (which are awful), plus some typo corrections and consistency fixes.

Doing a PR in case I messed up formatting.

### CODE REVIEW

File: data/language/nl-NL.txt [hunk 1]
```diff
@@ -5579,14 +5579,14 @@ STR_CPTY    :16 passagiers
 
 # Monorail Cycles / Fietsbaan
 [MONBK]
-STR_NAME    :Fietsbaan
+STR_NAME    :Fietsen
```
Comment (AaronVanGeffen) [inline data/language/nl-NL.txt]: Hmm, as there is a monorail track involved, I'd be inclined to keep the "baan" part in.

File: data/language/nl-NL.txt [hunk 2]
```diff
@@ -7272,6 +7532,10 @@ STR_NAME    :Basisblok
 #################
 # Large objects #
 #################
+
+[GENSTORE]
+STR_NAME    :General store
```
Comment (AaronVanGeffen) [inline data/language/nl-NL.txt]: Hmm… "Kruidenierswinkel"? Doesn't carry entirely the same meaning, though.

File: data/language/nl-NL.txt [hunk 3]
```diff
@@ -7393,7 +7676,7 @@ STR_NAME    :Houten schutting
 STR_NAME    :Junglehek
 
 [WMW]
-STR_NAME    :Muur in Marsstijl
+STR_NAME    :Buitenaardse muur
```
Comment (AaronVanGeffen) [inline data/language/nl-NL.txt]: English does refer to it as "Martial style wall", I presume?

File: data/language/nl-NL.txt (hunk 1 above)
Comment (Gymnasiast) [reply data/language/nl-NL.txt]: Dit is de voertuignaam. De naam van de attractie blijft 'fietsbaan'.

File: data/language/nl-NL.txt (hunk 2 above)
Comment (Gymnasiast) [reply data/language/nl-NL.txt]: Gaat hier specifiek om het Wildwestthema, vandaar deze "vertaling".

File: data/language/nl-NL.txt (hunk 3 above)
Comment (Gymnasiast) [reply data/language/nl-NL.txt]: Ja. Ik heb dit voornamelijk omwille van de consistentie veranderd.

**Pull request merged**